tabular/src/autogluon/tabular/trainer/abstract_trainer.py

```diff
@@ -380,20 +380,26 @@ def predict_proba(self, X, model=None):
             return self._predict_proba_model(X, model)
 
     # Note: model_pred_proba_dict is mutated in this function to minimize memory usage
-    def get_inputs_to_model(self, model, X, model_pred_proba_dict=None, fit=False, preprocess=True):
+    def get_inputs_to_model(self, model, X, model_pred_proba_dict=None, fit=False, preprocess=False):
+        """
+        For output X:
```

Comment:
clarify this explanation a bit more?

Reply:
This is a placeholder explanation, I will be adding far more documentation to abstract_trainer when I add repeated fit functionality.